apps/prairielearn/src/models/assessment.ts

```diff
@@ -66,6 +66,7 @@ export const AssessmentStatsRowSchema = AssessmentSchema.extend({
 export type AssessmentStatsRow = z.infer<typeof AssessmentStatsRowSchema>;
 
 export const AssessmentRowSchema = AssessmentStatsRowSchema.extend({
+  name: AssessmentSetSchema.shape.name,
```

Comment:
Why was this added to the schema and not removed from the query? Is this actually used anywhere?